app/views/admin/index.html.erb

```diff
@@ -59,9 +64,14 @@
     <%= render "admin/service", services: @services %>
   </section>
 
-  <section class="govuk-tabs__panel govuk-tabs__panel--hidden" id="teams">
-    <h2 class="govuk-heading-l">Teams</h2>
-     <%= render "admin/team", teams: @teams %>
+  <section class="govuk-tabs__panel govuk-tabs__panel--hidden" id="rp_teams">
+    <h2 class="govuk-heading-l">Relying Party Teams</h2>
+     <%= render "admin/team", teams: @relying_parties %>
+  </section>
+
+  <section class="govuk-tabs__panel govuk-tabs__panel--hidden" id="idp_teams">
+    <h2 class="govuk-heading-l">Identity Provider Teams</h2>
+     <%= render "admin/team", teams: @identity_providers %>
   </section>
```

Comment:
Seems this is repeated above can we maybe have this extracted into a partial and pass what changes as parameters